package protoFile.hall.FastEnterGroup;
message FastEnterGroupRequest{
     required int32 id=1;//½ÇÉ«id
}
message FastEnterGroupResponse{
     required bool result=1;
     optional string message=2;
}